### ISSUE TITLE
Add support for NewPipe's "Play with Kodi" option

### ISSUE THREAD

**Issue by wtimme** [petterreinholdtsen/kodi-invidious-plugin#6]
> **As a** user of NewPipe,
> **I want** Kodi to use the Invidious plugin when using "Play with Kodi",
> **so that** I can play and queue YouTube videos from within NewPipe on my phone

# Acceptance criterias

- When using the "Play with Kodi" button in NewPipe and then sending the video to the Kodi remote app "Kore", the Invidious plugin is opened in Kodi and starts playing the video

**Comment by wtimme** [comment]
Right now, when using "Play with Kodi" in NewPipe, Kodi reports

> To use this feature you must download an add-on:
> YouTube
> Would you like to download this add-on?

Unfortunately, the YouTube add-on for Kodi requires a dedicated API key, which needs to be obtained via the Google Developer Console. (Which, in turn, requires a Google account.)

By the way, thanks for this great plugin!

**Comment by wtimme** [comment]
So I did a bit of digging and had a look at the add-on's source. From what I understand, the add-on is already able to support this, at least the "Play on Kodi" option. (Queuing does not seem to be supported yet.)

The mechanism basically works like this:

1. NewPipe creates an intent with the YouTube URL that opens the Kodi remote app, Kore. NewPipe is unaware of any particular plugin that is being used within Kodi, and it does not have to be.
2. Kore processes the intent and parses the URL that was passed along. For YouTube links, it invokes the add-on `plugin.video.youtube` (see [here](https://github.com/xbmc/Kore/blob/389abee55a77dd630722cd7944f3ef550ca2bd48/app/src/main/java/org/xbmc/kore/ShareOpenActivity.java#L242)).

The Invidious add-on can be invoked like this:

```bash
wget \
--user $USER \
--password $PASSWORD \
http://$KODI_URL/jsonrpc \
--header="Content-Type: application/json" \
--post-data='{"jsonrpc": "2.0", "method": "Player.Open", "params": { "item":{"file":"plugin://plugin.video.invidious/?action=play_video&video_id=$VIDEO_ID" }},"id": 0 }'
```

(Of course, you'll need to replace `$USER`, `$PASSWORD`, `$KODI_URL` and `$VIDEO_ID` with the proper values.)

To sum it up, the place to add support for the Invidious plugin is not the add-on itself, but rather the Kore app for Android.

**Comment by wtimme** [comment]
I have created [a dedicated issue](https://github.com/xbmc/Kore/issues/978) for this feature request over at the repository of the Kore app, and I am going to close this issue here.

Feel free to re-open if you think that it needs to be tracked here.

Again, thanks for the add-on!
